server/shared/logger.ts

```diff
@@ -0,0 +1,20 @@
+import Logger, { LogLevel, Stream } from 'bunyan';
```

Comment:
@malizeus Have you guys considered moving away to `bunyan` to something more stable like `pino`?